### ISSUE TITLE
Game keeps opening in the background

### ISSUE THREAD

**Issue by Latatude** [ShadowMario/FNF-PsychEngine#16018]
### Describe your bug here.

Every time I launch PsychEngine.exe, it only appears in the background and not as an actual window. But sometimes it opens as intended. I don't know what's causing the difference.

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

No, I only reran `lime test cpp` to update the changes I made to make the base game assets part of the game, which already worked as intended the first time I launched the game right after the first compile.

**Comment by Shadow06735** [comment]
wdym appears in bg? u mean the app opens but u gotta click the icon again to actually open the app?

**Comment by Latatude** [comment]
> wdym appears in bg? u mean the app opens but u gotta click the icon again to actually open the app?

No, actually. When I click the icon, nothing pops up. But, after trying several times and a new window never pops up, when I check Task Manager, it shows a bunch of PsychEngine.exe processes listed under the "Background processes" section. Does that help explain things a bit?

**Comment by Latatude** [comment]
Why did @ShadowMario close the issues I've opened as completed if they weren't all properly completed and solved?
